examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -127,32 +124,42 @@ protected boolean matchesSafely(PipelineResult pResult) {
       Collection<String> files = factory.match(path);
 
       // Read data from file paths
+      int i = 0;
       for (String file : files) {
         try (Reader reader =
-                     Channels.newReader(factory.open(file), StandardCharsets.UTF_8.name())) {
-          readData.addAll(CharStreams.readLines(reader));
+              Channels.newReader(factory.open(file), StandardCharsets.UTF_8.name())) {
+          List<String> lines = CharStreams.readLines(reader);
+          readData.addAll(lines);
+          LOG.info(
+              "[{} of {}] Read {} lines from file: ", i, files.size() - 1, lines.size(), file);
```

Comment:
You seem to be missing the 4th argument which takes the file in the format string.

file: " -> file: {}"